### PR TITLE
[CI] Run Examples on Xcode 8 using CircleCI

### PR DESCRIPTION
**Pull request by DanToml**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -36,6 +36,7 @@ group :development do
   cp_gem 'molinillo',             'Molinillo'
   cp_gem 'nanaimo',               'Nanaimo'
   cp_gem 'xcodeproj',             'Xcodeproj'
+  cp_gem 'fourflusher',           'Fourflusher'
```
Comment (segiddins) [inline Gemfile]: this isn't in order

File: Rakefile [hunk 2]
```diff
@@ -295,10 +295,10 @@ begin
             when :osx
               execute_command "xcodebuild -workspace '#{workspace_path}' -scheme '#{scheme_name}' clean build"
             when :ios
-              # Specifically build against the simulator SDK so we don't have to deal with code signing.
-              # Need to use the iPhone 6s, since this is the default simulator paired with a watch in Xcode 7.3
               test_flag = (scheme_name.start_with? 'Test') ? 'test' : ''
-              execute_command "xcodebuild -workspace '#{workspace_path}' -scheme '#{scheme_name}' clean build #{test_flag} ONLY_ACTIVE_ARCH=NO -destination 'platform=iOS Simulator,name=iPhone 6s'"
+              watch_project = scheme_name.start_with? 'watchOS'
```
Comment (segiddins) [inline Rakefile]: unused variable

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+machine:
+  ruby:
+    version: system
+  xcode:
+    version: 8.1
+
+dependencies:
```
Comment (segiddins) [inline circle.yml]: does this prevent it from cloning the submodule?

File: Rakefile [hunk 4]
```diff
@@ -255,11 +255,11 @@ begin
   namespace :examples do
     desc 'Open all example workspaces in Xcode, which recreates the schemes.'
     task :recreate_workspace_schemes do
-      examples.each do |example|
+      Dir['examples/*'].each do |example|
```
Comment (DanToml) [inline Rakefile]: because examples was undefined

**Pull request merged**